src/home/homework.c

```diff
@@ -1909,6 +1963,13 @@ static int run(int argc, char *argv[]) {
         if (r < 0)
                 return r;
 
+        call_flags_env = getenv("SYSTEMD_HOMEWORK_CALL_FLAGS");
+        if (call_flags_env) {
```

Comment:
so if i get this right, we are not actually using the flags for anything yet, do we?

i think it's right to have a field for them in the new bus call. But I'd just check that they are zero early, and not process them any further, i.e. not propagate all the way down to homework. Let's do that once we actually need this here. I mean, the first flags we might eventually add could just be some we process in homed itself fully, without propagating them down...

Reply:
I already use this in #31153, and I need it propagated into homework... It would be rather annoying to rip this out here only to put it back in in the other PR.